metadata/Interstellar_platemap.csv

```diff
@@ -1,385 +1,385 @@
-F19,F19,6,19,11,Thapsi,10µM
-G19,G19,7,19,4,H2O2,50µM
-H19,H19,8,19,12,Thapsi,1µM
-I19,I19,9,19,5,ATP,1mM
-J19,J19,10,19,13,LPS + Nigericin,1µg/ml + 10µM
-K19,K19,11,19,6,ATP,0.1mM
-L19,L19,12,19,14,LPS + Nigericin,1µg/ml + 1µM
-M19,M19,13,19,7,Flagellin,1µg/ml
-N19,N19,14,19,15,DMSO 0.1%,0
-O19,O19,15,19,8,Flagellin,0.1µg/ml
-P19,P19,16,19,16,Media only,0
-A20,A20,1,20,7,Flagellin,1µg/ml
-B20,B20,2,20,15,DMSO 0.1%,0
-C20,C20,3,20,8,Flagellin,0.1µg/ml
-D20,D20,4,20,16,Media only,0
-E20,E20,5,20,1,LPS,10µg/ml
-F20,F20,6,20,9,Disulfiram,2.5µM
-G20,G20,7,20,2,LPS,1µg/ml
-H20,H20,8,20,10,Disulfiram,0.1µM
-I20,I20,9,20,3,H2O2,500µM
-J20,J20,10,20,11,Thapsi,10µM
-K20,K20,11,20,4,H2O2,50µM
-L20,L20,12,20,12,Thapsi,1µM
-M20,M20,13,20,5,ATP,1mM
-N20,N20,14,20,13,LPS + Nigericin,1µg/ml + 10µM
-O20,O20,15,20,6,ATP,0.1mM
-P20,P20,16,20,14,LPS + Nigericin,1µg/ml + 1µM
-A21,A21,1,21,8,Flagellin,0.1µg/ml
-B21,B21,2,21,16,Media only,0
-C21,C21,3,21,7,Flagellin,1µg/ml
-D21,D21,4,21,15,DMSO 0.1%,0
-E21,E21,5,21,6,ATP,0.1mM
-F21,F21,6,21,14,LPS + Nigericin,1µg/ml + 1µM
-G21,G21,7,21,5,ATP,1mM
-H21,H21,8,21,13,LPS + Nigericin,1µg/ml + 10µM
-I21,I21,9,21,4,H2O2,50µM
-J21,J21,10,21,12,Thapsi,1µM
-K21,K21,11,21,3,H2O2,500µM
-L21,L21,12,21,11,Thapsi,10µM
-M21,M21,13,21,2,LPS,1µg/ml
-N21,N21,14,21,10,Disulfiram,0.1µM
-O21,O21,15,21,1,LPS,10µg/ml
-P21,P21,16,21,9,Disulfiram,2.5µM
-A22,A22,1,22,14,LPS + Nigericin,1µg/ml + 1µM
-B22,B22,2,22,6,ATP,0.1mM
-C22,C22,3,22,13,LPS + Nigericin,1µg/ml + 10µM
-D22,D22,4,22,5,ATP,1mM
-E22,E22,5,22,12,Thapsi,1µM
-F22,F22,6,22,4,H2O2,50µM
-G22,G22,7,22,11,Thapsi,10µM
-H22,H22,8,22,3,H2O2,500µM
-I22,I22,9,22,10,Disulfiram,0.1µM
-J22,J22,10,22,2,LPS,1µg/ml
-K22,K22,11,22,9,Disulfiram,2.5µM
-L22,L22,12,22,1,LPS,10µg/ml
-M22,M22,13,22,16,Media only,0
-N22,N22,14,22,8,Flagellin,0.1µg/ml
-O22,O22,15,22,15,DMSO 0.1%,0
-P22,P22,16,22,7,Flagellin,1µg/ml
-A23,A23,1,23,8,Flagellin,0.1µg/ml
-B23,B23,2,23,7,Flagellin,1µg/ml
-C23,C23,3,23,6,ATP,0.1mM
-D23,D23,4,23,5,ATP,1mM
-E23,E23,5,23,4,H2O2,50µM
-F23,F23,6,23,3,H2O2,500µM
-G23,G23,7,23,2,LPS,1µg/ml
-H23,H23,8,23,1,LPS,10µg/ml
-I23,I23,9,23,16,Media only,0
-J23,J23,10,23,15,DMSO 0.1%,0
-K23,K23,11,23,14,LPS + Nigericin,1µg/ml + 1µM
-L23,L23,12,23,13,LPS + Nigericin,1µg/ml + 10µM
-M23,M23,13,23,12,Thapsi,1µM
-N23,N23,14,23,11,Thapsi,10µM
-O23,O23,15,23,10,Disulfiram,0.1µM
-P23,P23,16,23,9,Disulfiram,2.5µM
-A24,A24,1,24,16,Media only,0
-B24,B24,2,24,15,DMSO 0.1%,0
-C24,C24,3,24,14,LPS + Nigericin,1µg/ml + 1µM
-D24,D24,4,24,13,LPS + Nigericin,1µg/ml + 10µM
-E24,E24,5,24,12,Thapsi,1µM
-F24,F24,6,24,11,Thapsi,10µM
-G24,G24,7,24,10,Disulfiram,0.1µM
-H24,H24,8,24,9,Disulfiram,2.5µM
-I24,I24,9,24,8,Flagellin,0.1µg/ml
-J24,J24,10,24,7,Flagellin,1µg/ml
-K24,K24,11,24,6,ATP,0.1mM
-L24,L24,12,24,5,ATP,1mM
-M24,M24,13,24,4,H2O2,50µM
-N24,N24,14,24,3,H2O2,500µM
-O24,O24,15,24,2,LPS,1µg/ml
-P24,P24,16,24,1,LPS,10µg/ml
\ No newline at end of file
+well,wellName,row,col,alias,treatment,dose
```

Comment:
Should `wellName` data be the same as `well` data?

Reply:
Honestly, I don't think they need to be the same because I only needed one of those two columns to be corrected to contain "0" in them to be able to using the `annotate` function.